crates/virtio-queue/benches/queue/mod.rs

```diff
@@ -40,7 +38,7 @@ pub fn benchmark_queue(c: &mut Criterion) {
     let mem = GuestMemoryMmap::<()>::from_ranges(&[(GuestAddress(0x0), 0x1_0000_0000)]).unwrap();
 
     let queue_with_chains = |num_chains, len, indirect| {
-        let mut mq = MockSplitQueue::new(&mem, 256);
+        let mut mq = MockSplitQueue::new(GuestAddress(0), &mem, 256);
```

Comment:
Hmm, maybe it would be nicer and more convenient to have `new` always assume `GuestAddress(0)` as the start address, and have a separate constructor that allows us to specify a custom value.

Reply:
Done.